packages/smooth_app/lib/cache/files/files_cache_impl.dart

```diff
@@ -0,0 +1,129 @@
+// ignore_for_file: avoid_slow_async_io
+part of 'files_cache.dart';
```

Comment:
This is something I don't like in `dart`...

Reply:
Since the implementation is way simpler, everything is on a single file